src/main.py

```
@@ -1,232 +1,161 @@
-import_or_install("extension_mainclass")
-import_or_install("extension_themes")
-import_or_install("requirements")
-import_or_install("git_integration")
-import_or_install("settings")
-import_or_install("github")
-import_or_install("init_project")
-import_or_install("error_checker")
-import_or_install("merge_resolver")
-
-print("DEBUG: All dependencies successfully imported and initialized.")
+import extension_menubar
```

Ruff (0.11.9)

Module level import not at top of file

(E402)